distributed/nanny.py

```diff
@@ -259,9 +257,33 @@ async def start(self):
 
         await super().start()
 
-        await self.listen(
-            self._start_address, **self.security.get_listen_args("worker")
-        )
+        ports = parse_worker_ports(self._start_port)
+        for port in ports:
```

Comment:
Just for my understanding, does each nanny attempt to connect to the first port? One succeeds and the rest try the second? And so on?

Reply:
Yes, that's exactly what happens